tests/_.js

```diff
@@ -0,0 +1,318 @@
+import { assert } from 'chai';
+import * as _ from '../src/fe/implicit/_';
```

Comment:
And you'll need to update import paths after moving the test file to a different location.